docs/02_Preparing_Environment_for_a_VM_Cluster.md

```diff
@@ -682,25 +682,25 @@ A VM that was set up from a cloud image already has an SSH server up and running
 However, by default it is configured to reject login-based attempts. We must authenticate using a public key,
 which must be preconfigured on the VM.
 
-Make sure you have an SSH key prepared on the host machine (`~/.ssh/id_rsa.pub`).
+Make sure you have an SSH key prepared on the host machine (`~/.ssh/id_ed25519.pub`).
 If not, run:
 
 ```
 ssh-keygen
```

Comment:
Add the `-t ed25519` argument here to make sure this command does the same thing on every system (e.g. on current stable Debian, the default is still `rsa`)

Reply:
That works better!